From 40dc9d0bec1643c0812a9e5b9e3a7f3464f8c3a2 Mon Sep 17 00:00:00 2001
From: "Huang, Tao" <huangtao@rock-chips.com>
Date: Mon, 20 Feb 2017 15:58:42 +0800
Subject: [PATCH] arm64: dts: rk3399: rename android-next to android

The md5sum is identical after rename, so this commit is safe.

Change-Id: I97cb5faecebaad9d2e9c39f67f19f662642cc5e8
Signed-off-by: Huang, Tao <huangtao@rock-chips.com>
---
 arch/arm64/boot/dts/rockchip/Makefile              | 14 +++++++-------
 ...k3399-android-next.dtsi => rk3399-android.dtsi} |  0
 ...k3399-box-rev1-next.dts => rk3399-box-rev1.dts} |  2 +-
 ...k3399-box-rev2-next.dts => rk3399-box-rev2.dts} |  2 +-
 .../{rk3399-box-next.dtsi => rk3399-box.dtsi}      |  2 +-
 ...ndroid-next.dts => rk3399-evb-rev1-android.dts} |  2 +-
 ...ndroid-next.dts => rk3399-evb-rev2-android.dts} |  2 +-
 ...dp-next.dts => rk3399-evb-rev3-android-edp.dts} |  2 +-
 ...android-next.dts => rk3399-mid-818-android.dts} |  2 +-
 ...-next.dts => rk3399-sapphire-excavator-edp.dts} |  2 +-
 arch/arm64/boot/dts/rockchip/rk3399-tve1205g.dts   |  2 +-
 11 files changed, 16 insertions(+), 16 deletions(-)
 rename arch/arm64/boot/dts/rockchip/{rk3399-android-next.dtsi => rk3399-android.dtsi} (100%)
 rename arch/arm64/boot/dts/rockchip/{rk3399-box-rev1-next.dts => rk3399-box-rev1.dts} (99%)
 rename arch/arm64/boot/dts/rockchip/{rk3399-box-rev2-next.dts => rk3399-box-rev2.dts} (99%)
 rename arch/arm64/boot/dts/rockchip/{rk3399-box-next.dtsi => rk3399-box.dtsi} (99%)
 rename arch/arm64/boot/dts/rockchip/{rk3399-evb-rev1-android-next.dts => rk3399-evb-rev1-android.dts} (98%)
 rename arch/arm64/boot/dts/rockchip/{rk3399-evb-rev2-android-next.dts => rk3399-evb-rev2-android.dts} (98%)
 rename arch/arm64/boot/dts/rockchip/{rk3399-evb-rev3-android-edp-next.dts => rk3399-evb-rev3-android-edp.dts} (98%)
 rename arch/arm64/boot/dts/rockchip/{rk3399-mid-818-android-next.dts => rk3399-mid-818-android.dts} (99%)
 rename arch/arm64/boot/dts/rockchip/{rk3399-sapphire-excavator-edp-next.dts => rk3399-sapphire-excavator-edp.dts} (98%)

diff --git a/arch/arm64/boot/dts/rockchip/Makefile b/arch/arm64/boot/dts/rockchip/Makefile
index 8c69677edf09..23249851c34c 100644
--- a/arch/arm64/boot/dts/rockchip/Makefile
+++ b/arch/arm64/boot/dts/rockchip/Makefile
@@ -8,30 +8,30 @@ dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-box-rev1-android-6.0.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-box-rev1-disvr.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-box-rev2-android-6.0.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-box-rev2-disvr.dtb
-dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-box-rev1-next.dtb
-dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-box-rev2-next.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-box-rev1.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-box-rev2.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-disvr-android.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb-rev1-android-6.0.dtb
-dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb-rev1-android-next.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb-rev1-android.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb-rev1-cros.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb-rev2-android-6.0.dtb
-dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb-rev2-android-next.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb-rev2-android.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb-rev2-cros.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb-rev3-android-6.0.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb-rev3-android-6.0-edp.dtb
-dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb-rev3-android-edp-next.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb-rev3-android-edp.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-evb-rev3-cros.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-fpga.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-gru-gru.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-gru-kevin-r0.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-gru-kevin-r1.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-mid-818-android-6.0.dtb
-dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-mid-818-android-next.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-mid-818-android.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-rv1-android.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-sapphire-excavator-box-android-6.0.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-sapphire-excavator-disvr.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-sapphire-excavator-edp-android-6.0.dtb
-dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-sapphire-excavator-edp-next.dtb
+dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-sapphire-excavator-edp.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-sapphire-excavator-linux.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-tve1205g.dtb
 dtb-$(CONFIG_ARCH_ROCKCHIP) += rk3399-vr-android.dtb
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-android-next.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-android.dtsi
similarity index 100%
rename from arch/arm64/boot/dts/rockchip/rk3399-android-next.dtsi
rename to arch/arm64/boot/dts/rockchip/rk3399-android.dtsi
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-box-rev1-next.dts b/arch/arm64/boot/dts/rockchip/rk3399-box-rev1.dts
similarity index 99%
rename from arch/arm64/boot/dts/rockchip/rk3399-box-rev1-next.dts
rename to arch/arm64/boot/dts/rockchip/rk3399-box-rev1.dts
index 41683450e99c..03a864fa8e27 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-box-rev1-next.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-box-rev1.dts
@@ -41,7 +41,7 @@
  */
 
 /dts-v1/;
-#include "rk3399-box-next.dtsi"
+#include "rk3399-box.dtsi"
 
 / {
 	model = "Rockchip RK3399 Board rev1 (BOX)";
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-box-rev2-next.dts b/arch/arm64/boot/dts/rockchip/rk3399-box-rev2.dts
similarity index 99%
rename from arch/arm64/boot/dts/rockchip/rk3399-box-rev2-next.dts
rename to arch/arm64/boot/dts/rockchip/rk3399-box-rev2.dts
index 5f4c4c984d97..c2fdd7df1a6b 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-box-rev2-next.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-box-rev2.dts
@@ -41,7 +41,7 @@
  */
 
 /dts-v1/;
-#include "rk3399-box-next.dtsi"
+#include "rk3399-box.dtsi"
 
 / {
 	model = "Rockchip RK3399 Board rev2 (BOX)";
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-box-next.dtsi b/arch/arm64/boot/dts/rockchip/rk3399-box.dtsi
similarity index 99%
rename from arch/arm64/boot/dts/rockchip/rk3399-box-next.dtsi
rename to arch/arm64/boot/dts/rockchip/rk3399-box.dtsi
index abcab029ab55..3c2ca90d1524 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-box-next.dtsi
+++ b/arch/arm64/boot/dts/rockchip/rk3399-box.dtsi
@@ -42,7 +42,7 @@
 #include <dt-bindings/pwm/pwm.h>
 #include <dt-bindings/input/input.h>
 #include "rk3399.dtsi"
-#include "rk3399-android-next.dtsi"
+#include "rk3399-android.dtsi"
 #include "rk3399-opp.dtsi"
 
 / {
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev1-android-next.dts b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev1-android.dts
similarity index 98%
rename from arch/arm64/boot/dts/rockchip/rk3399-evb-rev1-android-next.dts
rename to arch/arm64/boot/dts/rockchip/rk3399-evb-rev1-android.dts
index c8e158ac223e..087042474103 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev1-android-next.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev1-android.dts
@@ -43,7 +43,7 @@
 /dts-v1/;
 
 #include "rk3399-evb-rev1.dtsi"
-#include "rk3399-android-next.dtsi"
+#include "rk3399-android.dtsi"
 
 / {
 	model = "Rockchip RK3399 Evaluation Board v1 (Android)";
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2-android-next.dts b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2-android.dts
similarity index 98%
rename from arch/arm64/boot/dts/rockchip/rk3399-evb-rev2-android-next.dts
rename to arch/arm64/boot/dts/rockchip/rk3399-evb-rev2-android.dts
index a5d69b2d22fe..31077f7f47d3 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2-android-next.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev2-android.dts
@@ -43,7 +43,7 @@
 /dts-v1/;
 
 #include "rk3399-evb-rev2.dtsi"
-#include "rk3399-android-next.dtsi"
+#include "rk3399-android.dtsi"
 
 / {
 	model = "Rockchip RK3399 Evaluation Board v2 (Android)";
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-edp-next.dts b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-edp.dts
similarity index 98%
rename from arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-edp-next.dts
rename to arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-edp.dts
index 840edcb4e7bd..3ada7aec8fe3 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-edp-next.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-evb-rev3-android-edp.dts
@@ -43,7 +43,7 @@
 /dts-v1/;
 
 #include "rk3399-evb-rev3.dtsi"
-#include "rk3399-android-next.dtsi"
+#include "rk3399-android.dtsi"
 
 / {
 	model = "Rockchip RK3399 Evaluation Board v3 edp (Android)";
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-mid-818-android-next.dts b/arch/arm64/boot/dts/rockchip/rk3399-mid-818-android.dts
similarity index 99%
rename from arch/arm64/boot/dts/rockchip/rk3399-mid-818-android-next.dts
rename to arch/arm64/boot/dts/rockchip/rk3399-mid-818-android.dts
index 30cd20d7b381..28906815a93c 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-mid-818-android-next.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-mid-818-android.dts
@@ -43,7 +43,7 @@
 /dts-v1/;
 
 #include "rk3399.dtsi"
-#include "rk3399-android-next.dtsi"
+#include "rk3399-android.dtsi"
 #include "rk3399-opp.dtsi"
 #include <dt-bindings/sensor-dev.h>
 #include <dt-bindings/pwm/pwm.h>
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-edp-next.dts b/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-edp.dts
similarity index 98%
rename from arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-edp-next.dts
rename to arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-edp.dts
index 1a312b4ff1bb..e8b0c9205726 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-edp-next.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-sapphire-excavator-edp.dts
@@ -43,7 +43,7 @@
 /dts-v1/;
 
 #include "rk3399-excavator-sapphire.dtsi"
-#include "rk3399-android-next.dtsi"
+#include "rk3399-android.dtsi"
 
 / {
 	model = "Rockchip RK3399 Excavator Board edp (Android)";
diff --git a/arch/arm64/boot/dts/rockchip/rk3399-tve1205g.dts b/arch/arm64/boot/dts/rockchip/rk3399-tve1205g.dts
index 57e7cf8da9c5..a16aae0c22cc 100644
--- a/arch/arm64/boot/dts/rockchip/rk3399-tve1205g.dts
+++ b/arch/arm64/boot/dts/rockchip/rk3399-tve1205g.dts
@@ -43,7 +43,7 @@
 /dts-v1/;
 #include <dt-bindings/pinctrl/rk.h>
 #include "rk3399.dtsi"
-#include "rk3399-android-next.dtsi"
+#include "rk3399-android.dtsi"
 #include "rk3399-opp.dtsi"
 #include <dt-bindings/sensor-dev.h>
 #include <dt-bindings/pwm/pwm.h>
-- 
2.35.3

